doc/man/scons.xml

```diff
@@ -331,6 +332,78 @@ if necessary. Each &ARGLIST; entry is a tuple containing
 (<replaceable>argname</replaceable>, <replaceable>argvalue</replaceable>).
 </para>
 
+<para>&scons;
+can maintain a cache of target (derived) files that can
+be shared between multiple builds.  When derived-file caching is enabled in an
+SConscript file, any target files built by
+&scons;
+will be copied
+to the cache.  If an up-to-date target file is found in the cache, it
+will be retrieved from the cache instead of being rebuilt locally.
+Caching behavior may be disabled and controlled in other ways by the
+<link linkend="opt-cache-force"><option>--cache-force</option></link>,
+<link linkend="opt-cache-disable"><option>--cache-disable</option></link>,
+<link linkend="opt-cache-readonly"><option>--cache-readonly</option></link>,
+and
+<link linkend="opt-cache-show"><option>--cache-show</option></link>
+command-line options.  The
+<link linkend="opt-random"><option>--random</option></link>
+option is useful to prevent multiple builds
+from trying to update the cache simultaneously.</para>
+
+<!--  The following paragraph reflects the default tool search orders -->
+<!--  currently in SCons/Tool/__init__.py.  If any of those search orders -->
+<!--  change, this documentation should change, too. -->
+
+<para>By default,
+&scons;
+searches for known programming tools
+on various systems and initializes itself based on what is found.
+On Windows systems which identify as <emphasis>win32</emphasis>,
+&scons;
+searches in order for the
+Microsoft Visual C++ tools,
+the MinGW tool chain,
+the Intel compiler tools,
+and the PharLap ETS compiler.
+On Windows system which identify as <emphasis>cygwin</emphasis>
+(that is, if &scons; is invoked from a cygwin shell),
+the order changes to prefer the GCC toolchain over the MSVC tools.
+On OS/2 systems,
+&scons;
+searches in order for the
+OS/2 compiler,
+the GCC tool chain,
+and the Microsoft Visual C++ tools,
+On SGI IRIX, IBM AIX, Hewlett Packard HP-UX, and Oracle Solaris systems,
+&scons;
+searches for the native compiler tools
+(MIPSpro, Visual Age, aCC, and Forte tools respectively)
+and the GCC tool chain.
+On all other platforms,
+including POSIX (Linux and UNIX) platforms,
+&scons;
+searches in order
+for the GCC tool chain,
+and the Intel compiler tools.
+These default values may be overridden
+by appropriate setting of &consvars;.</para>
+
+<para>&scons;
+requires Python 3.5 or higher.
```

Comment:
can 3.5 be variablized... ;) so we don't forget to update this?  Also didn't we just drop/deprecate 3.5?

Reply:
Might make sense to put the required python bit at the top of the manpage?

Reply:
> can 3.5 be variablized... ;) so we don't forget to update this?

Part of this isn't really easy, because we sometimes have a deprecated Python version and sometimes not. 

We can have something like `&pyminimum;`, but then we have to remember to update it in in `doc/scons.mod` (presuming that's where the entity would live) instead of in `doc/man/scons.xml`.  I'm not arguing against doing so, it just seems not to buy much, you still need a checklist item - unless we rig something to update it. docbook-xml doesn't have a good way to pull from `SCons/__init__.py` that I can think of.

Reply:
:( o.k. I guess simplest thing is to add to the checklist/procedure at: https://github.com/SCons/scons/wiki/SimplifiedReleaseProcedure

Reply:
I did add an item there to check, fwiw.